podcasts/podcasts.prototype.entitlements

```diff
@@ -11,7 +11,7 @@
 		<string>webcredentials:play.pocketcasts.com</string>
 		<string>applinks:pcast.pocketcasts.net</string>
 		<string>applinks:pca.st</string>
-		<string>applinks:pocketcasts.com</string>
+		<string>applinks:pocketcasts.com?mode=developer</string>
```

Comment:
Temporary change to allow us to test in real time changes in the apple-app-site-association file